recipe/meta.yaml

```diff
@@ -23,8 +22,8 @@ requirements:
     - cmake
     - make
   host:
-    - jasper >=2.0.14,<3  # [not ppc64le]
-    - openjpeg  # [ppc64le]
```

Comment:
This previously only used `openjpeg` on ppc64le systems, and did not use jasper in that case. Now this unconditionally depends on both. That seems wrong?

Reply:
i can revert. but that seems wrong too right?

Reply:
Well, I'd say looking at the build logs to understand what it's now doing on both platforms and seeing if it's correct or if one of them is unnecessary would be the right thing to do.

Reply:
ok, it seems that some previous tests were failing and it was explicitely disabled in the build. sorry for not tkaing a closer look.

Reply:
I think the next release of ecCodes will fix that failing test, or at least not run it with that specific combination of jasper+threads (am I right, @shahramn?)

Reply:
I've reverted the changes to openjpeg.